avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,29 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, count, option=None, sudo=False):
+        """This method is used to check packet loss that occurs during
+        ping to peer address.
+
+        Function returns True for 0% packet loss and False
+        if packet loss occurs.
+
+        :param peer_ip: Peer IP address (IPv4 or IPv6)
+        :param count: How many packets to send
+        :param option: ping command options. Default is None
+        :param flood: ping flood option -f. Default is False
+        """
+        cmd = f'ping -I {self.name} {peer_ip} -c {count}'
+        if sudo is not False:
+            cmd = f'{sudo} {cmd}'
```

Comment:
The sudo command should be handled by the `run_command(sudo=True)`. That will do exactly this, but respecting if the command is executed locally or remotely.